### PR TITLE
Added support for Threads share

### PR DESCRIPTION
**Pull request by WatermarkBranding**
Committing changes to src/ and build/, as well as updates to package-lock.json and readme. <VERSION> and <DATE> need to be updated appropriately in readme.txt.

### CODE REVIEW

**Comment by WatermarkBranding** [comment]
Hi! Hate to bump this, but I'd rather use your official plugin in production rather than manually installing. I know you're probably busy, but when you get a chance, please review. Thanks!

**Comment by ndiego** [comment]
Thanks for this PR @WatermarkBranding and sorry for the delay. I reviewed the update and made a few tweaks. Otherwise looks good and works as expected. Thanks for your contribution. This will be included in version 1.1.0, which will be out by Monday.

**Comment by WatermarkBranding** [comment]
No worries at all. Happy to contribute, and thanks for putting this together and making it available in the first place.

File: readme.txt [hunk 1]
```diff
@@ -65,11 +65,11 @@ Share options include:
 
 == Changelog ==
 
-= <VERSION> - <DATE> =
+= = 1.1.0 - <DATE> =
```
Comment (WatermarkBranding) [inline readme.txt:68]: May want to replace <DATE> here to with release date.

**Review by WatermarkBranding — COMMENTED** [review]
Minor: <DATE> should be replaced with release date

File: readme.txt (hunk 1 above)
Comment (ndiego) [reply readme.txt:68]: Yup thanks, the date is not finalized yet.

**Pull request merged**